### PR TITLE
Add maker user #141

### LINKED ISSUE #141: Maker Registration
Form needs following mandatory checkpoints to be checked for a maker to complete registration:

* Ich erkläre mich im Umfang meiner Möglichkeiten bereit, nach besten Möglichkeiten und Fähigkeiten kostenlos 3D-Drucke für Krankenhäuser und sonstige medizinische und soziale Einrichtungen sowie medizinisches Personal herzustellen, die auf dieser Internetpräsenz registriert sind.
* Regelungen zu etwaigen Materialkostenübernahmen sowie für etwaig anfallende Transportkosten werde ich mit den vermittelten Krankenhäusern und sonstigen medizinischen und sozialen Einrichtungen sowie medizinischem Personal unmittelbar treffen.
* Mir ist bekannt, dass print4health ausschließlich Kontakte zwischen Krankenhäusern und sonstigen medizinischen und sozialen Einrichtungen sowie medizinischem Personal einerseits und privaten 3D-Druckern und Designern von 3D-Druck-Bauplänen andererseits vermittelt.
* Ich erkläre mich daher damit einverstanden, dass print4health keinerlei Haftung
  * für das Zustandekommen von 3D-Druck-Aufträgen
und
  * für die Qualität der auf dieser Internetpräsenz vorgehaltenen 3D-Druck-Design-Vorlagen übernimmt.
* Einer Weitergabe der von mir mitgeteilten Kontaktdaten und Kapazitätsangaben durch print4health an registrierte Krankenhäuser und sonstige medizinische und soziale Einrichtungen sowie medizinisches Personal stimme ich ausdrücklich zu.

### PR DESCRIPTION
**Pull request by malteschlueter**
#141

### CODE REVIEW

**Review by gherkins — APPROVED** [review]

File: src/Domain/User/Entity/Maker.php [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+    public function getUsername(): string
+    {
+        return $this->email;
+    }
+
+    public function setEmail(string $email): self
+    {
+        $this->email = $email;
+
+        return $this;
```
Comment (DavidBadura) [inline src/Domain/User/Entity/Maker.php:105]: Except for a Builder, normal classes should not use a fluent interface. Here you can read why: https://ocramius.github.io/blog/fluent-interfaces-are-evil/

Comment (malteschlueter) [reply src/Domain/User/Entity/Maker.php:105]: Would you like to create an issue for this? :)

Comment (dragonito) [reply src/Domain/User/Entity/Maker.php:105]: Just remove it in this case. I´ll refactor the other code

Comment (DavidBadura) [reply src/Domain/User/Entity/Maker.php:105]: done #175

**Review by dragonito — APPROVED** [review]

**Pull request merged**